_posts/2020-02-07-rising-from-the-ashes.md

```diff
@@ -0,0 +1,46 @@
+---
+layout:     post
+title:      Rising from the Ashes
+date:       2020-02-07
+summary:    We've always enjoyed running incident response drills, but they were becoming stale. This post covers how we addressed the problems with our fire drills and iterated upon them
+author:     ols
+image:      firedrills/fire.jpg
+category:   Operations
+tags:       chaos engineering, fire drill
+---
+
+Just under two years ago, I wrote about [how we run fire drills (incident management role play scenarios) in the Core tribe](https://sbg.technology/2018/05/04/firedrills-in-core/). The post covered the mechanics of planning and running the drills, but also talked a bit about the tools that we were using. Since then, a lot has changed in how our fire drills work, and this post will cover the "what" and "why" of those changes. It is not necessary to read the previous article to understand the points in this one, but I would still recommend it so that you have additional context around what has changed.
```

Comment:
```suggestion
Just under two years ago, I wrote about [how we run fire drills (incident management role play scenarios) in the Core tribe](/2018/05/04/firedrills-in-core/). The post covered the mechanics of planning and running the drills, but also talked a bit about the tools that we were using. Since then, a lot has changed in how our fire drills work, and this post will cover the "what" and "why" of those changes. It is not necessary to read the previous article to understand the points in this one, but I would still recommend it so that you have additional context around what has changed.
```